browser/iframe-browser.js

```diff
@@ -0,0 +1,77 @@
+define(function (require, exports, module) {
+    "use strict";
+
+    var CommandManager      = brackets.getModule("command/CommandManager"),
+        MainViewManager     = brackets.getModule("view/MainViewManager"),
+        Commands            = brackets.getModule("command/Commands"),
+        _panel		        = null,
+        horizontal          = true,
+        vertical            = false;
```

Comment:
Don't manage this orientation state with two variables (the more variables you have, the more chance you have for bugs in your state logic).  Since these are mutually exclusive, you can simply have one.  Do something like this:

```js
var VERTICAL_ORIENTATION = 0;
var HORIZONTAL_ORIENTATION = 1;

// By default we use vertical orientation
var _orientation = VERTICAL_ORIENTATION;

// Expose these constants on our module, so callers can use them with setOrientation()
exports.HORIZONTAL_ORIENTATION = HORIZONTAL_ORIENTATION;
exports.VERTICAL_ORIENTATION = VERTICAL_ORIENTATION
```